templates/yymmdd-project-progress-report.md

```diff
@@ -21,12 +21,12 @@ __Roadmap amendments__
 
 _As of now, [CPS](https://cps.icon.community) projects may not request amendments to the originally proposed budget_
 
-_All budget amendment requests must provide strict and logical reasoning. The requester is susceptible to denial of request for budget amendment if the grant provider does not perceive this request to provide sufficient reason. Funding for CPS grants is available in bnUSD. Funding for ICON Foundation grants is available in ICX or USD_
+_All budget amendment requests must provide strict and logical reasoning. The requester is susceptible to denial of request for budget amendment if the grant provider does not perceive this request to provide sufficient reason. Funding for CPS grants is available in [bnUSD (stablecoin)](https://balanced.network/stablecoin/). Funding for ICON Foundation grants is available in ICX or USD_
 
 | Source | Amount | Time [days] | Notes |
 | ------ | ---------------------------- | ---- | ----- |
-| Person 1 | ICX 7000 | 10 days | This is the second developer. They will be necessary for an extra 10 days due to the change in expected completion date for Milestone 1 in the Roadmap Amendments |
```

Comment:
As budgets may be requested for different grants programs in ICX as well as in USD-based currencies, I think they should both be in here. Can you keep one of them as ICX and change the other to USD?

Reply:
Hmmm maybe it makes sense to keep this one as ICX then and the Project Proposal as USD?

Not sure if we support distribution of funds in two separate currencies for a single grant proposal.

Reply:
That was my original intent. One of them uses ICX and the other uses USD, so that people can see that they are both options